src/resource/resource-opening-hours/opening-period/PeriodOpeningHours.tsx

```diff
@@ -26,6 +26,38 @@ function findResourceStateLabelByValueAndLang(
   return resourceStateObj ? resourceStateObj[language] : null;
 }
 
+function renderPeriodStatusByLanguage(
+  periodStatus: ResourceState | undefined,
+  language: Language,
+  stateOptions: UiOptionsFieldConfig
+): JSX.Element {
+  if (periodStatus === 'open' || periodStatus === 'closed') {
```

Comment:
Im not sure if open and closed requires this prefix 'always'? What do you think? Should we extend the prefix to all status or remove it completely? Or leave it as it is? https://helsinkisolutionoffice.atlassian.net/browse/HAUKI-289

Reply:
I'm not sure do these even need prefix at all 🤔
We could leave it as it is and ask for more opinions.

Enums can be used in comparisons:
```js
 if (
    periodStatus === ResourceState.OPEN ||
    periodStatus === ResourceState.CLOSED
  ) {
```

Reply:
- [ ] Remove prefixes